packages/react-charting/src/components/PieChart/Arc/Arc.types.ts

```diff
@@ -30,6 +30,21 @@ export interface IArcProps {
    * The prop used to define the culture to localized the numbers
    */
   culture?: string;
+  /**
+   * to pass the theme
+   */
+  theme?: ITheme;
+}
+
+export interface IArcState {
+  /**
+   * The state controls, whether the arc needs to be focused or not
+   */
+  isArcFocused?: boolean;
+  /**
+   * Unique Identifier for each arc
+   */
+  id: string;
```

Comment:
id need not be a state variable. It can be a class variable.

Reply:
I see, fixing this